src/Analyzers/CSharp/Tests/ValidateFormatString/ValidateFormatStringTests.cs

```diff
@@ -544,33 +537,35 @@ static void Main(string[] args)
 ");
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.ValidateFormatString)]
-        public async Task WarningTurnedOff()
```

Comment:
`WarningTurnedOff` was broken in both the C# and VB tests. I have combined it with `WarningTurnedOn` to execute on the same source but with different `expectDiagnostic` values.